general/app/development/links/deep-linking.md

```diff
@@ -91,3 +91,4 @@ moodlemobile://link=https://mysite.es/mod/choice/view.php?id=8
 ## See also
 
 - [Custom URL Scheme Cordova plugin used by the app](https://github.com/EddyVerbruggen/Custom-URL-scheme).
+- [Links in the Moodle App](./links.md)
```

Comment:
Is it necessary to add this, if they are both in the same section? I guess the sidebar should be fairly visible, even in mobile... So yeah I don't know, I wouldn't add it but I guess it's ok if you want to.

Reply:
Right, I removed it.